files/setup.sh

```diff
@@ -8,55 +8,177 @@
+    messageManualOverclock="Manual Edit 'nvidia-overclock.sh' (For Experts)"
+    messageManualMining="Manual Edit 'miner.sh' (For Experts)"
+    messageDetectTitle="Detect and Generate Monitoring Sensors"
+    messageSensorsOk="Done! After a reboot, you can use the command 'sensors' to monitor the sensors."
+    
+    #Large Messages
+    
+    lmessage1MineETH="With this ISO image, you can immediately mine Ethereum (ETH). Do not spend long time searching and researching."
+    lmessage1Happy="I would be happy about a small donation. Thank you very much."
+    lmessage1ETH="Ξ - Ethereum: 0xfbbc9f870bccadf8847eba29b0ed3755e30c9f0d"
+    lmessage1BTC="Ƀ - Bitcoin:  13fQA3mCQPmnXBDSmfautP4VMq6Sj2GVSA"
+    
+    lmessage2SensorsTxt="Detect and generate a list of kernel modules for monitoring temperatures, voltage, and fans."
+    lmessage2SensorsSafe="The programm 'sensors-detect' will ask to probe for various hardware. The safe answers are the defaults, so just hitting [Enter] to all the questions will generally not cause any problems."
+}
+	
+#Mexican Spanish
+
+es_MX () {
+	#Estado
+	
+	languageStatus=1
+
+	#Mensajes
+	
+	messageFailure="FALLO"
+	messagePressAnyKey="Presione cualquier tecla para continuar..."
+}
+
+#####################################################################
+# Aliases
+#####################################################################
+
+#gkho64: Clear is great for quick screen cleanups, BUT, if you have ever seen above, it DOES NOT clear the screen at all, because if you scroll up your terminal (In case of being running the script in a terminal managed by a WM), you'll see all the evidence (Except if "Dialog" takes care of all that trash...).
+#gkho64: Another native alternative would be to use "reset", but it's kinda slow clearing the screen, but at least, it gets the job correctly done.
+#gkho64: As a third option, we have "tput reset", which does the same, but this one is faster cleaning up the screen, leaving no trace of any previous text. The only counterpart of it, is that, actually, it isn't part of Ubuntu itself nor "coreutils" package, so Ubuntu Server (Or any other distro) may not come preinstalled with it. To workaround that, just make sure to install "ncurses" package. I'll leave that decision to you...
+
+selectScreenCleaner () {
+	if [ "$1" -eq 0 ]; then
+		alias clearScreen="clear"
+	elif [ "$1" -eq 1 ]; then
+		alias clearScreen="reset"
+	elif [ "$1" -eq 2]; then
+		alias clearScreen="tput reset"
+	else
+		exit 46
+	fi
+}
+
+#####################################################################
+# Startup Functions
+#####################################################################
+
+#gkho64: First, let's load stablish a "Screen Cleaner" and load "Default Symbols"
+
+selectScreenCleaner 0
+defaultSymbols
+
+#gkho64: Then, let's load a "Language Pack" (For sake of clarity, let's call it like that)
+	#gkho64: By default, en_US will be loaded
+	
+	en_US
```

Comment:
My opinion only makes sense if the language is recognized automatically. All users should also be able to read English. Otherwise the user would not have found the image :-)  Complicates everything...